Sources/SwiftSyntaxBuilderGeneration/main.swift

```diff
@@ -0,0 +1,199 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import Foundation
+import SwiftSyntax
+import SwiftSyntaxBuilder
+
+private let format = Format(indentWidth: 2)
+
+private let copyrightHeader = """
+    //// Automatically Generated
```

Comment:
```suggestion
    //// Automatically Generated by SwiftSyntaxBuilderGeneration
```

Just so we know it’s not generated by `.gyb` files and keep time spending looking for them.